src/Tickets/Seating/app/frontend/session/index.js

```diff
@@ -417,10 +419,6 @@ async function getInterruptDialogElement() {
  * @return {void} The timer is interrupted.
  */
 async function interrupt() {
-	if (!isInterruptable()) {
```

Comment:
Here and elsewhere in this file: why remove this? I understand WooCommerce integration requires the session to not be interrupted when the user changes page, but is the Tickets Commerce integration still working as intended?

Reply:
Yes, I have tested myself and also QA is testing TicketsCommerce to make sure it is working just like before.